### ISSUE TITLE
Maximize button of Puddletag window disappearing

### ISSUE THREAD

**Issue by JoeJoeTV** [puddletag/puddletag#768]
### Current Behavior

When I start Puddletag, the maximize button is (most of the time) there and I can maximize the window. If I then un-maximize the window, it's suddenly gone and I can only resize it manually. 

### Expected Behavior

The window should have a maximize button all the time and can be maximized and un-maximized.

### Steps To Reproduce

1. Open Puddletag
2. Maximize and un-maximize it
3. Button is gone

### Information about your system

Puddletag: 2.2.0
OS: Linux Mint 21 Cinnamon
Python: 3.10.6
PyQt: 5.15.6

**Comment by JoeJoeTV** [comment]
Hi, thanks for the reply.

To clarify, I'm running version 2.2.0 installed as a deb I got from the Ubuntu 22.10 repos, which did install normally and I have 2 monitors side by side(1680x1050, 1920x1080(main)).
I have also already tried executing puddletag from the terminal and with -v and have not seen any additional logging, when the button disappears.
Also, I found out that it disappears directly after maximizing the first time.

I'm gonna try to narrow it down, but I think it has to do something, with selecting different windows inside puddletag to show.

**Comment by JoeJoeTV** [comment]
Yes, if I delete the `~/.config/puddletag` folder it starts maximized, but when sonfiguring something, which I'm trying to figure out, the maximize button disappears after minimizing and maximizing.

**Comment by JoeJoeTV** [comment]
I think, I've found the culprit:
When I start of a fresh config and start puddletag on my second monitor and then continue to select the windows "Tag Sources", "Functions" and "Actions"(order doesn't matter), the window switches to the main monitor and resizes to be too big for the second one(without bottom panel) and also loses the maximize button.
I then can't make the window smaller than 1040 pixels(excluding header bar and menu bar) and can't maximize it in any way(either button or by right clicking in the panel)

**Comment by JoeJoeTV** [comment]
And I just notices, that when I disable one of the mentioned windows, I can one again change the height of the window and the maximize button comes back.

**Comment by sandrotosi** [comment]
this is a tricky one. first off, i would prefer if we can find a way not to define a default screen size (1920x1080px or anything else) since we really dont know screen resolutions/fonts/magnifications etc used by users.

im just wondering here: is the only side effect is the maximize button being disable once the extra panels are added to the main window and that gets over the current screen size? if that's the case, and i understand how that may sound un-cool, maybe it's just a thing we need to live with and do nothing about?

**Comment by sandrotosi** [comment]
i re-read your previous comments and id like to know what does your gut feeling tell you :) what do you feel would be the right approach here?
